src/sentry/static/sentry/app/views/settings/components/forms/formField/formField.jsx

```diff
@@ -1,72 +1,26 @@
-import {Box, Flex} from 'grid-emotion';
 import {Observer} from 'mobx-react';
-import {css} from 'emotion';
 import PropTypes from 'prop-types';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import styled, {keyframes} from 'react-emotion';
 
-import {defined} from '../../../../utils';
-import FormState from '../../../../components/forms/state';
-import IconCheckmarkSm from '../../../../icons/icon-checkmark-sm';
-import IconWarningSm from '../../../../icons/icon-warning-sm';
-import Spinner from './styled/spinner';
-
-const SettingsPanelItemWrapper = styled(({highlighted, ...props}) => <Flex {...props} />)`
-  padding: 15px 20px;
-  border-bottom: 1px solid ${p => p.theme.borderLight};
-  align-items: center;
-  transition: background 0.15s;
-
-  ${p =>
-    p.highlighted
-      ? css`
-          outline: 1px solid ${p.theme.purple};
-        `
-      : ''} &:last-child {
-    border-bottom: none;
-  }
-`;
-
-const SettingsPanelItemLabel = styled.div`
-  color: ${p => p.theme.gray5};
-`;
-
-const SettingsPanelItemCtrl = styled(Box)`
-  color: ${p => p.theme.gray3};
-  width: 50%;
-  padding-left: 10px;
-  position: relative;
-`;
-
-const SettingsPanelItemCtrlState = styled(Box)`
-  width: 36px;
-  text-align: right;
-`;
-
-const SettingsPanelItemDesc = styled(Box)`
-  width: 50%;
-  padding-right: 10px;
-`;
-
-const SettingsRequiredBadge = styled.div`
-  display: inline-block;
-  background: ${p => p.theme.gray2};
-  width: 5px;
-  height: 5px;
-  border-radius: 5px;
-  text-indent: -9999em;
-  vertical-align: super;
-`;
-
-const SettingsPanelItemHelp = styled.div`
-  color: ${p => p.theme.gray2};
-  font-size: 14px;
-  margin-top: 8px;
-  line-height: 1.4;
-`;
-
-const SettingsErrorReason = styled.div`
+import {defined} from '../../../../../utils';
+import FormState from '../../../../../components/forms/state';
+import {
+  FormFieldCtrl,
+  FormFieldCtrlState,
+  FormFieldDesc,
+  FormFieldHelp,
+  FormFieldLabel,
+  FormFieldRequiredBadge,
+  FormFieldWrapper,
+} from './index';
```

Comment:
I think we should avoid doing this because it adds some mental overhead (when is it a default vs named export?). "default" exports have some issues too (e.g. you can alias a default import) but we almost always use default exports right now. I know it sux having to import each one individually but we should try to be consistent.